packages/paste-core/components/form/stories/index.stories.tsx

```diff
@@ -0,0 +1,425 @@
+              ))}
+            </Select>
+          </FormControl>
+          <FormControl>
+            <Label htmlFor={seed('zip')}>Zip code</Label>
+            <Input id={seed('zip')} type="text" />
+          </FormControl>
+        </FormControlTwoColumn>
+      </FormSection>
+      <FormActions>
+        <Button variant="primary">Save</Button>
+        <Button variant="secondary">Cancel</Button>
+      </FormActions>
+    </Form>
+  );
+};
+
+export const HorizontalForm: StoryFn = () => {
+  const seed = useUIDSeed();
+  return (
+    <Form aria-labelledby={seed('form-heading')}>
+      <Heading id={seed('form-heading')} as="h3" variant="heading30" marginBottom="space0">
+        {/* Added UID to the heading because forms are HTML Landmarks. Axe was failing because in the stacked view, there were 4 forms with the same name and landmarks have to have unique names. */}
+        Choose Network Access Profile {useUID()}
+      </Heading>
+      <Paragraph marginBottom="space0">
+        Network Access Profile specifies what networks your SIM&apos;s would connect to and how you will be billed.
+      </Paragraph>
+      <FormControl>
+        <CheckboxGroup name="number-type" legend="Select number type" orientation="horizontal">
+          <Checkbox>Local</Checkbox>
+          <Checkbox>Mobile</Checkbox>
+          <Checkbox>Toll-free</Checkbox>
+        </CheckboxGroup>
+      </FormControl>
+      <FormControl>
+        <RadioGroup name="payment-method" legend="Select payment method" orientation="horizontal">
+          <Radio>Credit card</Radio>
+          <Radio>PayPal</Radio>
+          <Radio>Direct deposit</Radio>
+        </RadioGroup>
+      </FormControl>
+      <FormActions>
+        <Button variant="primary">Save</Button>
+        <Button variant="secondary">Cancel</Button>
+      </FormActions>
+    </Form>
+  );
+};
+
+export const ErrorForm: StoryFn = () => {
+  const seed = useUIDSeed();
+
+  return (
+    <Form aria-labelledby={seed('form-heading')}>
+      <Heading id={seed('form-heading')} variant="heading30" as="h3">
+        {/* Added UID to the heading because forms are HTML Landmarks. Axe was failing because in the stacked view, there were 4 forms with the same name and landmarks have to have unique names. */}
+        Settings {useUID()}
+      </Heading>
+      <Callout variant="error">
+        <CalloutHeading as="h4">Please fix the issues with these fields</CalloutHeading>
+        <CalloutList as="ul">
+          <CalloutListItem>Street address</CalloutListItem>
+          <CalloutListItem>Email address</CalloutListItem>
+        </CalloutList>
+      </Callout>
+      <FormControl>
+        <Label htmlFor={seed('street-address')} required>
+          Street address
+        </Label>
+        <Input id={seed('street-address')} type="text" hasError required />
+        <HelpText variant="error">Please add a valid street address.</HelpText>
+      </FormControl>
+      <FormControlTwoColumn>
```

Comment:
suggestion: it might be nice to show what an error would look like in a two column form control

Reply:
added!